src/api/db/migrate/20210205113812_squasher_clean.rb

```diff
@@ -0,0 +1,12 @@
+class SquasherClean < ActiveRecord::Migration[6.0]
+  class SchemaMigration < ApplicationRecord
+  end
+
+  def up
+    migrations = Dir.glob(File.join(File.dirname(__FILE__), '*.rb'))
+    versions = migrations.map { |file| File.basename(file)[/\A\d+/] }
+    SchemaMigration.where.not(version: versions).delete_all
```

Comment:
we clean old migrations from the schema that don't exist in the file system anymore.